Assets/Scripts/SlashCollisions.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+public class SlashCollisions : MonoBehaviour
+{
+    float slashLifetime; 
+
+    void Start()
+    {
+        slashLifetime = 0.2f;
```

Comment:
Out of curiosity, how this slashLifetime is used here?
And why it is not used for BulletCollisions?

Reply:
Bullets exist until they hit something, slashes exist for a set amount of time, therefore this timer is required only for melee. slashLifetime is just how long a slash stays on screen. In this case, after every slash a timer for it is created.

Reply:
Okey, thanks for explanation